ci-operator/step-registry/hypershift/mce/agent/create/agentserviceconfig/hypershift-mce-agent-create-agentserviceconfig-commands.sh

```diff
@@ -70,7 +70,10 @@ END
 }
 
 function deploy_mirror_config_map() {
-  oc get configmap -n openshift-config user-ca-bundle -o json | jq -r '.data."ca-bundle.crt"' | awk '{ print "    " $0 }' > /tmp/ca-bundle-crt
+  if [ "${DISCONNECTED}" = "true" ]; then
+    oc get configmap -n openshift-config user-ca-bundle -o json | \
+      jq -r '.data."ca-bundle.crt"' | awk '{ print "    " $0 }' > /tmp/ca-bundle-crt
+  fi
```

Comment:
Why change it?

Reply:
See this particular commit for explanation: https://github.com/openshift/release/pull/63516/commits/f1605cdce4e6a3ac6b5ede8cc34635ee167583bf

Reply:
I needee to disable local registry via `ENABLE_LOCAL_REGISTRY=` here https://github.com/openshift/release/pull/63516/files#diff-928b817c787c81d69fccee73df9dd02c823917b4b80c31c82a40554cdcbb60c3R60 
That's to workaround the issue with pushing images in devscripts to the internal registry (I'm preparing a fix for devscripts but that might come later). Devscripts can't push the images that are specified by DIGESTS.
And when local registry is disabled devscripts don't install the "user-ca-bundle" ConfigMap. 
We only need the ConfigMap for disconnected anyway.